epinorm/config.py

```diff
@@ -55,8 +54,15 @@ def get_work_dir() -> Path:
 SCRIPT_DIR = PACKAGE_DIR / "scripts"
 REF_DATA_DIR = PACKAGE_DIR / "data"
 WORK_DIR = get_work_dir()
-COUNTRIES_DATA = PACKAGE_DIR / "data" / "countries.csv"
-ADMIN_LEVELS_DATA = PACKAGE_DIR / "data" / "administrative_units.tsv"
+
+HOST_SPECIES_FILE = REF_DATA_DIR / "ncbi_host_species.csv"
+PATHOGEN_SPECIES_FILE = REF_DATA_DIR / "ncbi_pathogen_species.csv"
+
+COUNTRIES_FILE = PACKAGE_DIR / "data" / "countries.csv"
+ADMIN_LEVELS_FILE = PACKAGE_DIR / "data" / "administrative_units.tsv"
```

Comment:
Rename the ADMIN_LEVELS_FILE constant to ADMIN_UNITS_FILE (to avoid confusion) and update all references to it.

```suggestion
ADMIN_UNITS_FILE = REF_DATA_DIR / "administrative_units.tsv"
```